dev-docs/decision-records/cve.md

```diff
@@ -0,0 +1,41 @@
+# Title [ADR-5]
+
+# CVE
+
+## Status
+
+Proposed
+
+## Context
+
+Deciding if a bug is CVE worthy needs definition so that we can be consistent.
+
+A CVE will result in the following being triggered:
+1. The creation of the CVE itself.
+2. Apollo support will reach out to all customers giving with a CVE giving them a heads up.
+3. A retro to discuss how the CVE happened in the first place, and how it was handled.
+
+The Router is a critical piece of infrastructure for our customers, and we need to be careful about how we handle security issues.
+However, at this time we still need to be pragmatic about what we consider CVE worthy. In particular, while denial of service is a top priority, in practice not possible to prevent at this time.
```

Comment:
```suggestion
However, at this time we still need to be pragmatic about what we consider CVE worthy. In particular, while denial of service is a top priority, in practice it is not possible to prevent these right now.
```